src/styles/blocks/product-cards.scss

```diff
@@ -0,0 +1,109 @@
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.product-card {
+  box-sizing: border-box;
```

Comment:
better 

```
html {
  box-sizing: border-box;
}

*,
*::after,
*::before {
  box-sizing: inherit;
}
```

Reply:
and move on all reset styles (box-sizing: border-box; and margin: 0; padding: 0;) to `reset.scss`